operators/operator_sdk/osdk-new-scorecard.adoc

```diff
@@ -0,0 +1,358 @@
+tests are designed for resource creation.
+
+[id="osdk-new-scorecard-configuration"]
+== Scorecard configuration
+
+The scorecard test execution is driven by a configuration file typically named
+`config.yaml`. The configuration file is located at the following
+location within your bundle:
+
+[source,text]
+----
+tests/scorecard/config.yaml
+----
+
+[id="osdk-new-scorecard-configuration-file"]
+=== Configuration file
+
+A sample of the scorecard configuration file may look as follows:
+
+[source,yaml]
+----
+tests:
+- name: "basic-check-spec"
+  image: quay.io/operator-framework/scorecard-test:dev
+  entrypoint:
+  - scorecard-test
+  - basic-check-spec
+  labels:
+    suite: basic
+    test: basic-check-spec-test
+  description: check the spec test
+- name: "olm-bundle-validation"
+  image: quay.io/operator-framework/scorecard-test:dev
+  entrypoint:
+  - scorecard-test
+  - olm-bundle-validation
+  labels:
+    suite: olm
+    test: olm-bundle-validation-test
+  description: validate the bundle test
+----
+
+[TIP]
+====
+See _Complete scorecard configuration file_ for an example that is used for
+running the scorecard pre-defined tests that ship with the Operator SDK.
+====
+
+The configuration file defines each test that scorecard can execute. The
+following fields of the scorecard configuration file define the test as follows:
+
+[cols="3,7",options="header"]
+|===
+|Configuration field |Description
+
+|`image`
+|The test container image name that implements a test
+
+|`entrypoint`
+|The command and arguments that are invoked in the test image to execute a test
+
+|`labels`
+|Scorecard-defined or custom labels that select which tests to run
+|===
+
+[id="osdk-new-scorecard-cmd-args"]
+=== Command arguments
+
+The scorecard command has the following syntax:
+
+[source,terminal]
+----
+$ operator-sdk alpha scorecard <bundle_path> | <bundle_path_image_name> <flags>
+----
+
+Scorecard requires a positional argument that holds either the on-disk path to
+your Operator bundle or the name of a bundle image.
+
+For further information about the flags, run:
+
+[source,terminal]
+----
+$ operator-sdk alpha scorecard -h
+----
+
+[id="osdk-new-scorecard-selecting-tests"]
+== Selecting tests
+
+Tests are selected by setting the `--selector` CLI flag to a set of label
+strings. If a selector flag is not supplied, then all the tests within the
+scorecard configuration file are executed.
+
+Tests are executed serially, one after the other, with test results
+being aggregated by scorecard and written to standard output (stdout).
+
+To select a single test, for example `basic-check-spec-test`:
+
+[source,terminal]
+----
+$ operator-sdk scorecard -o text --selector=test=basic-check-spec-test
+----
+
+To select a suite of tests, `olm` in this case, you can specify a label that is
+used by all the OLM tests:
+
+[source,terminal]
+----
+$ operator-sdk scorecard -o text --selector=suite=olm
```

Comment:
@adellape it should be "operator-sdk alpha scorecard ./ -o text --selector=suite=olm"